src/blocks/venue-information/save.js

```diff
@@ -0,0 +1,76 @@
+import { useBlockProps } from '@wordpress/block-editor';
+
+import { Icon } from '@wordpress/components';
+
+import MapEmbed from '../../helpers/map-embed';
+
+export default function save({ attributes }) {
+	const { mapId, fullAddress, phoneNumber, website, zoom, type, deskHeight } =
+		attributes;
+
+	const blockProps = useBlockProps.save();
+
+	return (
+		<div {...blockProps}>
+			{fullAddress && (
+				<>
+					<address className="gp-venue__full-address-row">
+						<span className="gp-venue__icon">
+							<Icon icon="location" />
+						</span>
+						<span>
+							{fullAddress && (
+								<div className="gp-venue__full-address">
+									{fullAddress}
+								</div>
+							)}
+						</span>
+					</address>
+				</>
+			)}
+			{fullAddress && (
+				<>
+					<div className="gp-venue__map">
+						<MapEmbed
+							location={fullAddress}
+							zoom={zoom}
+							type={type}
+							height={deskHeight}
+							className={`embed-height_${mapId}`}
+						/>
+					</div>
+				</>
+			)}
```

Comment:
Match rendering of `edit.js` Add address to a `gp-venue__row` and add the map to the bottom of the black. save and edit rendering should look the same.

Reply:
Thinking more about this... I this it's better to go back to the `render.php` here because changes to `save.js` could bring breaking changes to the block (Block was broken when I checked out your branch in editor). An alternative way to do this is store the data for the map in the venue markup in `data-` attributes, then include `venue-information.js` file and include it as `viewScript` attribute in `block.json`. The `add-to-calendar` block is an example of how you can do this.